integr8ly/cloud-resource-operator
# force apply any pending modifications on service upates for redis tha…

## Overview

Jira: <jira-id>

## Verification

- On a cluster where there is a redis which has the service update in pending modification

- Verify that the serviceUpdate entry with the offending service update is present in aws-cloud-resource-strategies config map.
- Verify that the redis cr contains the applyImmediately flag as true
- Check the `serviceUpdateApplyDate` is before time.Now()
- If all of the above are satisfied, 
- update the image to `quay.io/austincunningham/cloud-resource-operator:v0.29.0`

You should see logs similar to this -> 
```
	"Checking status of Update Action = %s ", *update.ServiceUpdateName)
	"UpdateActionStatus = %s ", *update.UpdateActionStatus)
         "ServiceUpdateSeverity = %s ", *update.ServiceUpdateSeverity)
	"ServiceUpdateStatus = %s ", *update.ServiceUpdateStatus)
```

You should also see the redis going into a modifying state
And after some time the UpdateActionStatus of the clsuter should no longer be `update available` but `up to date`




## Checklist
